examples/StandardFirmataWiFi/StandardFirmataWiFi.ino

```diff
@@ -309,6 +309,12 @@ void checkDigitalInputs(void)
 }
 
 // -----------------------------------------------------------------------------
+// function forward declarations
+void enableI2CPins();
```

Comment:
I see, you are compiling via the command line rather than using the Arduino IDE.